### ISSUE TITLE
disabled button?

### ISSUE THREAD

**Issue by tofutim** [zeroclipboard/zeroclipboard#636]
I'm trying to use ZeroClipboard from cdn and running into a lot of issues. I finally boiled it down to the simple example but in this example the button gets disabled? Why?

```
<html>
<head>
<script src="https://cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.js"></script>

</head>
<body>
    <button id="copy-button" data-clipboard-text="Copy Me!" title="Click to copy me.">Copy to Clipboard</button>
<script>
ZeroClipboard.config( { swfPath: "https://cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.swf" } );
var client = new ZeroClipboard( document.getElementById("copy-button") );

client.on( "ready", function( readyEvent ) {
  // alert( "ZeroClipboard SWF is ready!" );

  client.on( "aftercopy", function( event ) {
    // `this` === `client`
    // `event.target` === the element that was clicked
    event.target.style.display = "none";
    alert("Copied text to clipboard: " + event.data["text/plain"] );
  } );
} );
</script>
</body>
</html>

```

**Comment by tofutim** [comment]
Maybe because I am running the page from the desktop?

**Comment by aoles** [comment]
See: https://github.com/zeroclipboard/zeroclipboard/blob/master/docs/instructions.md#cross-protocol-limitations

TL;DR: Try using ZeroClipboard.js and ZeroClipboard.swf hosted locally, or at jsDelivr: http://www.jsdelivr.com/#!zeroclipboard

**Comment by tofutim** [comment]
I had to use jsDelivr - I think there is a problem with the one from cdnjs.
I also notice that the latest (github) version switched from cdnjs to
jsDelivr. It now also works without specifying the swfPath.

**Comment by JamesMGreene** [comment]
Duplicate of #632
